examples/run-iteration/README.md

```diff
@@ -0,0 +1,3 @@
+# Hello World Example
+
+To execute run the following on the root directory of the repository: `cargo run --example helloworld`.
```

Comment:
in case this example stays:
```suggestion
# Run Iteration Example

To execute run the following on the root directory of the repository: `cargo run --example run-iteration`.
```